src/cluster.c

```diff
@@ -2404,7 +2480,7 @@ int clusterProcessPacket(clusterLink *link) {
                     {
                         serverLog(LL_VERBOSE,
                             "Node %.40s has old slots configuration, sending "
-                            "an UPDATE message about %.40s",
+                            "an UPDATE message about %.40s ",
```

Comment:
This is the main use case I was thinking about for node name. You can print it out here so that the logs have information that is useful to humans.